<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Thumbnails_Size_DockWidget</class>
 <widget class="QDockWidget" name="Thumbnails_Size_DockWidget">
  <property name="windowTitle">
   <string>Thumbnails Size</string>
  </property>
  <widget class="QWidget" name="Thumbnails_Size_dockWidgetContents">
   <layout class="QGridLayout" name="Thumbnails_Size_dockWidgetContents_gridLayout">
    <item row="0" column="0">
     <widget class="QLabel" name="Smallest_Size_label">
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QLabel" name="Largest_Size_label">
      <property name="minimumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>24</width>
        <height>24</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QSlider" name="Thumbnails_Size_horizontalSlider">
      <property name="statusTip">
       <string>Thumbnails Size Horizontal Slider : Adjust Database Browser Icon Size.</string>
      </property>
      <property name="minimum">
       <number>24</number>
      </property>
      <property name="maximum">
       <number>128</number>
      </property>
      <property name="value">
       <number>128</number>
      </property>
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <spacer name="Thumbnails_Size_verticalSpacer">
      <property name="orientation">
       <enum>Qt::Vertical</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>20</width>
        <height>40</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
